sunpy/cm/__init__.py

```diff
@@ -1,7 +1,9 @@
-"""
-The following colormaps are provided by this module:
-"""
```

Comment:
It's a bit odd that this package no longer has a docstring.  How about importing `sunpy.visualization.colormaps` and copying over the `__doc__`?